src/adagrams.js

```diff
@@ -1,8 +1,142 @@
+import _, {
+  findLastIndex
+} from "underscore"
+
 const Adagrams = {
   drawLetters() {
     // Implement this method for wave 1
+    const letterPool = [];
+    const addLetters = function (letters, frequency) {
+      letters.forEach(function (letter) {
+        for (let i = 0; i < frequency; i++) {
+          letterPool.push(letter);
+        }
+      })
```

Comment:
As an arrow function, this would look like this: 
```suggestion
    const addLetters = (letters, frequency) => {
      letters.forEach(function (letter) {
        for (let i = 0; i < frequency; i++) {
          letterPool.push(letter);
        }
      })
```